openreview/arr/management/setup_shared_data.py

```diff
@@ -242,6 +242,7 @@ def _is_not_available(month: str, year, current_date: datetime.datetime, posted_
                 )
 
     # Conditionally post unavailability notes
+    senior_area_chairs_id = reviewers_id = domain.content['senior_area_chairs_id']['value'].replace(venue_id, previous_cycle_id)
```

Comment:
Doesn't look like we use `reviewers_id` anywhere after this, but I think this is a typo? Should we remove `reviewers_id` here?